recipes/z5py/build.sh

```diff
@@ -0,0 +1,67 @@
+# Platform-specific dylib extension
+if [ $(uname) == "Darwin" ]; then
+    export CC=clang
+    export CXX=clang++
+    export DYLIB="dylib"
+else
+    export CC=gcc
+    export CXX=g++
+    export DYLIB="so"
+fi
+
+##
+## START THE BUILD
+##
+
+mkdir -p build
+cd build
+
+CXXFLAGS="${CXXFLAGS} -I${PREFIX}/include"
+LDFLAGS="${LDFLAGS} -Wl,-rpath,${PREFIX}/lib -L${PREFIX}/lib"
+
+if [ $(uname) == Darwin ]; then
+    CXXFLAGS="$CXXFLAGS -stdlib=libc++"
+fi
+
+# Set the correct python flags, depending on the distribution
+PY_VER=$(python -c "import sys; print('{}.{}'.format(*sys.version_info[:2]))")
+PY_ABIFLAGS=$(python -c "import sys; print('' if sys.version_info.major == 2 else sys.abiflags)")
+PY_ABI=${PY_VER}${PY_ABIFLAGS}
+
+##
+## Configure
+##
+cmake .. \
+        -DCMAKE_C_COMPILER=${CC} \
+        -DCMAKE_CXX_COMPILER=${CXX} \
+        -DCMAKE_BUILD_TYPE=RELEASE \
+        -DCMAKE_OSX_DEPLOYMENT_TARGET=10.9\
```

Comment:
We set the macOS deployment target [here]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/d1dd6b5aeb3349c91079b6078d844760a788f302/recipe/conda_build_config.yaml#L39-L40 ). We also set the corresponding `CMAKE_OSX_DEPLOYMENT_TARGET` environment variable as part of the compilers. So this could either be dropped or set to one of those two environment variables, whichever you see fit. :)